pom.xml

```diff
@@ -4,7 +4,7 @@
 
     <groupId>es.tid.fiware</groupId>
     <artifactId>keypass</artifactId>
-    <version>1.2.1</version>
+    <version>1.3.0</version>
```

Comment:
It has been a long time since the last Keypass change, so I don't know if we were following the standard versioning procedures here. Are you going to release a new version right now? Maybe this should be labeled as 1.3.0-snapshot